src/main/java/no/nav/tag/finnkandidatapi/kandidat/KandidatRepository.java

```diff
@@ -53,7 +53,7 @@ public KandidatRepository(JdbcTemplate jdbcTemplate, SimpleJdbcInsert simpleJdbc
     public Optional<Kandidat> hentNyesteKandidat(String aktørId) {
         try {
             Kandidat kandidat = jdbcTemplate.queryForObject(
-                    "SELECT * FROM kandidat WHERE (aktor_id = ?) ORDER BY registreringstidspunkt DESC LIMIT 1", new Object[]{aktørId},
+                    "SELECT * FROM kandidat WHERE (aktor_id = ?) ORDER BY id DESC LIMIT 1", new Object[]{aktørId},
```

Comment:
@andrmos Er dette ok?

Reply:
Gode spørsmål, jeg er litt usikker.

Jeg har prøvd å google litt og det beste svaret jeg fant var dette: [Performance of max() vs ORDER BY DESC + LIMIT 1](https://stackoverflow.com/a/34259643/3238798)
Hvis jeg tolker kommentaren der riktig så ser det ut som at siden vi bruker `serial` typen på ID, så vil det genereres en sekvens satt med `NON NULL` ([Postgres doc](https://www.postgresql.org/docs/9.4/datatype-numeric.html#DATATYPE-SERIAL)), så `ORDER BY` vs. `max()` vil da ikke ha noen betydning.

Hva tenker du?

Reply:
Vil performance her ha noe å si for oss på stadiet vi er på nå egentlig?

Reply:
Synes den spørringen ser helt fin ut, og sortering bør fungere så lenge feltet man sorterer på ikke har NULL-verdier, som @andrmos nevner.
Jeg tror heller ikke det vil være noen merkbar forskjell i ytelse nesten uansett hva man gjør, så lenge _WHERE_-clausen reduserer resultatsettet før man sorterer. Da ender man jo i praksis opp med et sett (før sortering) som i de aller fleste tilfeller vil inneholde < 10 rader, skulle jeg tro. 
Hvis sortering skjer _før_ filtreringen i _WHERE_ clausen ville ytelsen kanskje hatt noe å si. Men jeg er usikker på om det vil være noen forskjell på max() vs ORDER BY uansett, da jeg antar at databasen må scanne alle de utvalgte radene for å finne max()? Med mindre den vedlikeholder en indeks som alltid har en oppdatert max()-verdi pr. bruker.